homeassistant/components/overkiz/sensor.py

```diff
@@ -92,7 +92,7 @@ class OverkizSensorDescription(SensorEntityDescription):
         name="Water volume estimation at 40 °C",
         icon="mdi:water",
         native_unit_of_measurement=UnitOfVolume.LITERS,
-        device_class=SensorDeviceClass.VOLUME,
+        device_class=SensorDeviceClass.WATER,
```

Comment:
This is the device class available for this right now:

```suggestion
        device_class=SensorDeviceClass.VOLUME_STORAGE,
```

Reply:
How that's a nice addition. Thx!